tegel/src/components/data-table/table-body-row/table-body-row.tsx

```diff
@@ -32,10 +35,27 @@ export class TableBodyRow {
     this.runPaginationEvent.emit(this.uniqueTableIdentifier);
   }
 
+  connectedCallback() {
+    this.tableEl = this.host.closest('sdds-table');
+    this.enableMultiselect =
+      !(this.tableEl.getAttribute('enable-multiselect') === 'false') &&
+      this.tableEl.hasAttribute('enable-multiselect');
```

Comment:
```suggestion
    this.enableMultiselect = this.tableEl.enableMultiselect;
```
I prefer this, because it's easier to read and has typescript typesafety